scripts/ingests/21Legg_ingest.py

```diff
@@ -0,0 +1,401 @@
+    coord = SkyCoord(ra_sexg, dec_sexg)
+    ra = coord.ra.degree
+    dec = coord.dec.degree
+
+    return ra, dec
+
+
+def add_publications(db):
+    ingest_publication(db, publication='Meis20.74', bibcode='2020ApJ...889...74M')  # Meisner_2020a
+    ingest_publication(db, publication='Meis20.123', bibcode='2020ApJ...899..123M')  # Meisner_2020b
+    ingest_publication(db, bibcode='2020ApJ...895..145B')
+    ingest_publication(db, bibcode='2021ApJS..253....7K')
+    ingest_publication(db, bibcode='2019ApJ...881...17M')
+    ingest_publication(db, bibcode='2021ApJ...918...11L')
+    ingest_publication(db, bibcode='2020ApJ...889..176F')
+    ingest_publication(db, publication='Pinf14.priv',
+                       description='P. Pinfield and M. Gromadzki, private communication 2014')
+
+    update_ref_dict = {
+        "Burn10": "Burn10.1952",
+        "Burn10b": "Burn10.1885",
+        "Burn11": "Burn11.3590",
+        "Burn11b": "Burn11.90",
+        "Burg00a": "Burg00.57",
+        "Burg00b": "Burg00.473",
+        "Burg00c": "Burg00.1100",
+        "Burg02a": "Burg02.421",
+        "Burg02b": "Burg02.2744",
+        "Burg02c": "Burg02.151",
+        "Burg03d": "Burg03.510",
+        "Burg03b": "Burg03.512",
+        "Burg03c": "Burg03.1186",
+        "Burg03e": "Burg03.2487",
+        "Burg03a": "Burg03.850",
+        "Burg04c": "Burg04.73",
+        "Burg04a": "Burg04.827",
+        "Burg04b": "Burg04.2856",
+        "Burg04d": "Burg04.191",
+        "Burg06c": "Burg06.1095",
+        "Burg06e": "Burg06.585",
+        "Burg06b": "Burg06.1067",
+        "Burg06a": "Burg06.1007",
+        "Burg06d": "Burg06.1485",
+        "Cush11": "Cush11.50",
+        "Delo08a": "Delo08.961",
+        "Delo08b": "Delo08.469",
+        "Dupu15a": "Dupu15.102",
+        "Dupu15b": "Dupu15.56",
+        "Geli11": "Geli11.57",
+        "Geli11b": "Geli11.871",
+        "Lodi09b": "Lodi09.1631",
+        "Lodi09d": "Lodi09.258",
+        "Lodi07a": "Lodi07.1423",
+        "Lodi07b": "Lodi07.372",
+        "Lodi12a": "Lodi12.1495",
+        "Lodi12b": "Lodi12.105",
+        "Lodi12d": "Lodi12.53",
+        "Loop07a": "Loop07.1162",
+        "Loop07b": "Loop07.97",
+        "Liu_11a": "Liu_11.32",
+        "Liu_11b": "Liu_11.108",
+        "Luhm12": "Luhm12.135",
+        "Luhm12d": "Luhm12.152",
+        "Luhm14b": "Luhm14.18",
+        "Luhm14d": "Luhm14.16",
+        "Luhm14a": "Luhm14.4",
+        "Luhm14c": "Luhm14.126",
+        "Luhm14e": "Luhm14.6",
+        "Mace13": "Mace13.6",
+        "Mace13b": "Mace13.36",
+        "Pinf14": "Pinf14.1931",
+        "Pinf14a": "Pinf14.1009",
+        "Scho10a": "Scho10.92",
+        "Scho10b": "Scho10.8",
+        "Tinn05a": "Tinn05.1171",
+        "Tinn05b": "Tinn05.2326",
+        "Warr07": "Warr07.1400",
+        "Warr07a": "Warr07.213",
+    }
+
+    for old_ref, new_ref in update_ref_dict.items():
+        db.Publications.update().where(db.Publications.c.publication == old_ref).values(
```

Comment:
For the future: it may have been good to have the updating of publications separate from the ingest, perhaps as part of the existing update_publications.py file and in its own branch. A lot of the JSON files changed only because the publication designation changed. That being said, there's nothing wrong with having it here if the publications being addressed directly relate to the Legget ingest.